src/main/kotlin/net/atos/zac/identity/IdentityService.kt

```diff
@@ -60,4 +67,22 @@ class IdentityService @Inject constructor(
             .map { it.toUser() }
             .sortedBy { it.getFullName() }
     }
+
+    fun checkIfUserIsInGroup(userId: String?, groupId: String?) {
+        if (userId == null || groupId == null) {
+            LOG.fine { "Missing user id '$userId' or group id '$groupId'. Cannot check is user is in group!" }
+            return
+        }
+
+        val isUserInGroup = listUsersInGroup(groupId)
+            .map { it.id }
+            .contains(userId)
+
+        if (!isUserInGroup) {
+            LOG.warning(
+                "User '$userId' is not in group '$groupId' and can therefore not be set as case worker."
```

Comment:
'set as case worker' is not in the context of this function. 
```suggestion
                "User '$userId' is not in group '$groupId'."
```